C/ds/Dijsktra.c

```diff
@@ -0,0 +1,119 @@
+/*	Dijkstra's Algorithm and code in C    */
+#include<stdio.h>
+#include<stdlib.h>
+
+void main(){
+
+	int cost[10][10],distance[10],path[10][10],n,v,p,row,column,min,index=1,i,j;
```

Comment:
Use Dynamic Values

Reply:
done @siddharth25pandey 
kindly review it